src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -152,8 +152,8 @@ const WorkflowIOTable = ({ which, inputsOutputs: data, config, errors, onChange,
         cellRenderer: ({ rowIndex }) => {
           const io = sortedData[rowIndex]
           const { name, optional, inputType } = io
-          const value = config[which][name] || ''
-          const error = errors[which][name]
+          const value = _.get([which, name], config) || ''
```

Comment:
The language now supports the following

```suggestion
          const value = config?.[which]?.[name] || ''
```

which may read more easily when you don't need to programmatically build the key path or need currying provided by `_.get`.

If you like this suggestion, please also update the line below accordingly.

Note that [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) also takes care of the error cases like Lodash does.

Reply:
huh, I tried that and it complained about syntax. Lemme give that another shot, I agree it's nicer to read …

Reply:
dunno what I did wrong the first time, optional chaining works as expected, as you suggested

Reply:
Glad it worked. I feel l end up needing to look up the order of `?` and `.` every other time  😊